src/components/UI/organisms/MenuItem/MenuItem.css

```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+:root {
+  --main: #fad390;
+  --secondary: #636e72;
+}
+
+.MenuItem {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: 1fr repeat(2, 0.5fr) repeat(2, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  background-color: var(--main);
+  align-items: center;
+  margin: 1rem;
```

Comment:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO decides in the project overall are you using `px` or `rem` - using them in parallel makes it in one point fight with each other about priority. Only `one` main format for the whole project - not less, not more.